### ISSUE TITLE
tf networks -- batch size could not be modified in loaded model

### ISSUE THREAD

**Issue by nicolay-r** [nicolay-r/AREkit#16]
Possible modifiable parameters are:
dropout,
embedding_dropout

for opimizer:
learning rate

**Comment by nicolay-r** [comment]
Google BERT at this point adopt: `TPUEstimator`, which allows to modify batch size due to wrapping such parameter in the related estimator ctor:
https://www.tensorflow.org/api_docs/python/tf/compat/v1/estimator/tpu/TPUEstimator

**Comment by nicolay-r** [comment]
Or, another way is to provide `None` instead of batch sizes in placeholder shapes (1-st dimention)
https://groups.google.com/a/tensorflow.org/g/discuss/c/ELSMzF3ucR4
